pyproject.toml

```diff
@@ -16,6 +16,8 @@ singer-sdk = "^0.4.7"
 types-simplejson = "^3.17.2"
 types-python-dateutil = "^2.8.6"
 nested-lookup = "^0.2.23"
+bs4 = "^0.0.1"
+types-beautifulsoup4 = "^4.11.1"
```

Comment:
If these are optional as you suggest in the scraping file, maybe they could be specified as such? https://python-poetry.org/docs/pyproject/#extras

Reply:
actually, I think the types package should be dev-dependencies instead.

Reply:
Making it optional seems complicated with an opt-in. I moved the type to "dev" but I think we can keep beautifulsoup4 like this.